designs/administrator/future-openshift/cluster-settings/index.md

```diff
@@ -0,0 +1,64 @@
+- Three new alerts will be added to inform users if (1) a new patch becomes available, (2) a new minor release becomes available, and/or (3) new channels become available. 
+- All three messages send users to the cluster settings section of the console to take further action.
+
+## Channel Visualization and In Context Help
+
+The new channel visualization provides users with more transparency into recommended upgrade paths, informs users what changes happen in each release, and guides users to the correct channel. 
+
+In addition to the visualization: 
+- Field level help text was added for channels
+- A new manage subscription field was added with a direct link to manage subscriptions in OpenShift Cluster Manager
+
+![Channel help text](img/channel-help.png)
+- Clicking `Channel` surfaces a popover that provides context as to what a channel is along  with a link to documentation to learn more about channels in OpenShift.
+
+![Channel visualization](img/channel-visualization.png)
+- The blue line signifies the current channel you are on. The current version and recommended version (if available) are both marked with blue dots. 
+- The gray line signifies the next available channel.
+- To surface recommended versions in the next available channel, users must switch to the channel by clicking the `Pencil` next to `Channel`. 
+- If no other channels are available the gray line will not be visible. 
+- At this time the visualization only supports surfacing one additional channel - the next available.
+
+![Channel visualization - recommended version popover](img/recommended-channel.png)
+- Clicking on the recommended version, in this case the `4.3.22 dot`, will surface a popover with recommended minor release information and a direct link to release notes.
+
+![Channel visualization - “+ more” popover](img/more-channels.png)
+- Clicking `+ More` will surface a popover that lists out all versions (between the current version and recommended version) in a table with direct links to release notes. 
+
+![Channel visualization - “+ more” modal](img/modal.png)
+- In the case that there are more than 5 versions available (between the current version and recommended version), clicking `+ More` will surface a modal.
+
+## Updating a Channel and Viewing the In Progress Checklist
+
+It was important to provide more transparency to the user when an update is in progress. The mockups below illustrate the steps a user takes to update to a minor release in their current channel, and surfaces the in progress checklist that allows users to see the status of the update in real time.
+
+![Update to minor release](img/update.png)
+- Clicking `Update` launches a modal to confirm the update.
+
+![Update modal](img/update-modal.png)
+- The recommended version is preselected. 
+- If available, users can select an alternative version in the dropdown.
+- Clicking `Update` will begin the update.
+
+![In progress check list](img/check-list.png)
```

Comment:
did you consider using the finite steps progress bar here? https://www.patternfly.org/v4/documentation/core/components/progress#finite-step
It seems like a nice fit but I might be missing something. Nice to use a PF thing that's already available when possible.

Reply:
@matthewcarleton Just took a look at that pattern and I think that could be a good alternative! I am not sure how possible it is to add the icon in the spot where it lists the finite steps above the progress bar though. I do think it would be a better use of space if it is possible!

Reply:
@matthewcarleton 
The only real downfall I see of using the component strictly as is, is we lose the context of the count. 

@beanh66 @gdoyle1 Any thoughts on how important it is to have the context of the count once that particular portion is complete?

These are the two options side by side. 

<img width="772" alt="Screen Shot 2020-07-01 at 2 55 11 PM" src="https://user-images.githubusercontent.com/62031580/86281281-4cc4f780-bbab-11ea-9a6c-03d00323a79f.png">
<img width="736" alt="Screen Shot 2020-07-01 at 2 56 10 PM" src="https://user-images.githubusercontent.com/62031580/86281286-4df62480-bbab-11ea-97d7-551f6fe93dad.png">

Reply:
@megan-hall Definitely in favor of using option 1!

Reply:
@megan-hall ya that's a good point. I wonder if that's something we could push back to PF for a future enhancement? It seems like something that other products could benefit from too.